pkgs/by-name/si/simulide/package.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, stdenv
+, fetchbzr
+, libsForQt5
+}:
+
+let
+  version = "0.4.15";
+  release = "SR10";
+  branch = "simulide_0.4.14"; # the branch name does not mach the version for some reason
+  rev = "291";
+  sha256 = "sha256-BBoZr/S2pif0Jft5wrem8y00dXl08jq3kFiIUtOr3LM=";
```

Comment:
what if you use `hash` instead of `sha256`?

Reply:
I believe there isn't a `hash` option for `fetchbzr`. (Maybe I could add it)
I searched around on git a bit more, and it looks like some people are downloading archives from launchpad. It looks like there are sources released [here](https://launchpad.net/simulide/0.4.15/0.4.15-stable) for *this* version.
Sadly, not all tags have the sources released in an archive, so I think that that `fetchbzr` will be more universal.

Reply:
As @TomaSajt  said apparently there isn't a `hash` option `fetchbzr`.  I tried to use `hash` when we migrate from appImage to compiling everything from source, but I ran into this problem.

Reply:
Ok, fair enough. You can mark it as resolved then.